### DISCUSSION TITLE
Displaying important stats in a minimalistic fashion

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#2144] [Q&A] [answered]
Hi! I need to display stats on a dashboard. I am using `ui.stats()` within a form card. However, using icons etc. is making it look a bit too much with respect to the over all dashboard. I am looking to display something like this:

![image](https://github.com/h2oai/wave/assets/117831190/bc43cfd2-6b13-47ae-9e65-e5c63fa3322e)

Is it possible to change font colours in Wave?
Font colours within `ui.table()` as well as other inidividual components?I am looking to change colours of text in each column of a `ui.table()` based on value.

**Comment by mturoci** [comment, marked as answer]
You can try something like this:

```py
ui.text('<span style='color: red'>Hello</span>')
```

This should work as ui.text is interpreted as markdown which allows for arbitrary HTML.

**Reply by n-srinidhi** [reply]
Thanks a lot !I'll try this.
